src/core/dropdownmenu/dropdownmenu.component.ts

```diff
@@ -0,0 +1,42 @@
+import {CommonModule} from "@angular/common";
+import {Component, EventEmitter, Input, Output} from "@angular/core";
+
+@Component({
+  selector: "wisdom-dropdownmenu",
+  imports: [CommonModule],
+  templateUrl: "./dropdownmenu.component.html",
+  styles: ``,
+})
+export class DropdownmenuComponent {
+  /**
+   * module name, when no name applied -> Test
+   */
+  @Input() menuName: string = "Test";
+
+  /**
+   * default options if no array is submitted.
+   */
+  @Input() options: string[] = ["A", "B", "C"];
+
+  /**
+   * event emitter submitting chosen option to parent component
+   */
+  @Output() choiceChange = new EventEmitter<string>();
+
+  /**
+   * chosen option, empty if none is chosen yet
+   */
+  choice!: string;
```

Comment:
Use a signal for the `choice`: https://angular.dev/essentials/signals

Then we won't need the event anymore. Also the type of the should be `string | null` as the initial value is empty. `!` tells the tsc that the value isn't undefined, but here it is.